packages/cli/scripts/v1-v2.js

```diff
@@ -57,7 +57,11 @@ function convertYAML(inputYAML) {
                     delete data.integrations[integration][taskName];
                 } else if (task.runs) {
                     change.type = 'sync';
-                    task.output = task.returns ? task.returns[0] : null;
+                    if (Array.isArray(task.returns)) {
+                        task.output = task.returns.length === 1 ? task.returns[0] : task.returns;
```

Comment:
why would we care about returning `[0]` if we can return `task.returns` anyway?
I feel like the fix is `task.output = task.returns`

Reply:
Well, output shouldn't be a single array element but rather a single element. Otherwise if it is multi model then only should it be an array

Reply:
Talked about on this notion I believe: https://www.notion.so/nangohq/Nov-2023-nango-yaml-Revamp-4755d2717a0142dfb9a447e37eb91a5e?pvs=4

Reply:
ah I didn't know there was a diff between single model vs multi model 👌🏻